gcp_variant_transforms/libs/annotation/vep/vep_runner_test.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import unittest
+
+import logging
+import mock
+
+from mock import patch
+
+from apache_beam.io import filesystems
+from gcp_variant_transforms.libs.annotation.vep import test_util
+from gcp_variant_transforms.libs.annotation.vep import vep_runner
+
+
+_OUTPUT_DIR = 'gs://output/dir'
+_VEP_INFO_FIELD = 'TEST_FIELD'
+_IMAGE = 'gcr.io/image'
+_CACHE = 'path/to/cache'
+_NUM_FORK = 8
+_PROJECT = 'test-project'
+_REGION = 'test-region'
+
+
+class _MockFileSystems(filesystems.FileSystems):
+  """This inherits from FileSystems such that most functions behave the same."""
+
+  @staticmethod
+  def match(patterns, limits=None):
+    if len(patterns) == 1 and patterns[0] == test_util.INPUT_PATTERN:
+      return [mock.Mock(
+          metadata_list=[test_util.FileMetadataStub(path, size) for
+                         (path, size) in test_util.INPUT_FILES_WITH_SIZE])]
+    return []
+
+  @staticmethod
+  def create(path, mime_type=None, compression_type=None):
+    """Overriding `create` to remove any interaction with real file systems."""
+    return mock.Mock()
+
+
+class VepRunnerTest(unittest.TestCase):
+
+  def _create_test_instance(self, pipeline_args=None):
+    self._mock_service = mock.Mock()  # pylint: disable=attribute-defined-outside-init
+    self._mock_pipelines = mock.Mock()  # pylint: disable=attribute-defined-outside-init
+    self._mock_request = mock.Mock()  # pylint: disable=attribute-defined-outside-init
+    self._mock_service.pipelines = mock.Mock(return_value=self._mock_pipelines)
+    self._mock_pipelines.run = mock.Mock(return_value=self._mock_request)
+    self._mock_request.execute = mock.Mock(return_value={'name': 'operation'})
+    test_object = vep_runner.VepRunner(
+        self._mock_service, test_util.INPUT_PATTERN, _OUTPUT_DIR,
+        _VEP_INFO_FIELD, _IMAGE, _CACHE, _NUM_FORK,
+        pipeline_args or self._get_pipeline_args())
+    return test_object
+
+  def _get_pipeline_args(self, num_workers=1):
+    return ['--project', _PROJECT,
+            '--region', _REGION,
+            '--max_num_workers', str(num_workers),
+            '--worker_machine_type', 'n1-standard-8',
+           ]
+
+  def test_instantiation(self):
+    """This is just to test object construction."""
```

Comment:
I'm curious if you expect this to actually catch something that's not covered by other tests? If not, I suggest removing it as it's trivial.